content/docs/guide/map_submission/fixing_entities.md

```diff
@@ -0,0 +1,349 @@
+---
+title: Fixing Entities and Other Issues
+categories:
+  - guide
+tags:
+  - mapping
+  - guidelines
+weight: 4
```

Comment:
I think the nav link to this page should come immediately after Map Porting, since it is used after that page but before zoning. If the docs system we use allows more nav sublevels, it might be nice to have a "Map Porting" subgroup with the main Map Porting page and this page grouped together.